client/signup/steps/woocommerce-install/select-site/index.tsx

```diff
@@ -0,0 +1,82 @@
+import {
+	isFreePlan,
+	isPremiumPlan,
+	isPersonalPlan,
+	isEcommercePlan,
+	isBusinessPlan,
+} from '@automattic/calypso-products';
+import { Card } from '@automattic/components';
+import { useI18n } from '@wordpress/react-i18n';
+import { ReactElement, useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import SiteSelector from 'calypso/components/site-selector';
+import StepWrapper from 'calypso/signup/step-wrapper';
+import { submitSignupStep } from 'calypso/state/signup/progress/actions';
+import { getSiteSlug } from 'calypso/state/sites/selectors';
+import type { WooCommerceInstallProps } from '../';
+
+import './style.scss';
+
+interface Site {
+	capabilities: {
+		manage_options: boolean;
+	};
+	jetpack: boolean;
+	plan: {
+		product_slug: string;
+	};
+}
+
+export default function SelectSite( props: WooCommerceInstallProps ): ReactElement | null {
+	const { goToStep, isReskinned } = props;
+	const { __ } = useI18n();
+
+	const filterSites = ( site: Site ) => {
+		return (
+			site.capabilities.manage_options &&
+			( isFreePlan( site.plan.product_slug ) ||
+				isPersonalPlan( site.plan.product_slug ) ||
+				isPremiumPlan( site.plan.product_slug ) ||
+				isBusinessPlan( site.plan.product_slug ) ||
+				isEcommercePlan( site.plan.product_slug ) )
```

Comment:
p1638330066305700-slack-C02DQP0FP not enough to block p2 sites - they're free plans, need to also check they're not wpforteams